staging/src/k8s.io/client-go/util/exec/exec.go

```diff
@@ -24,6 +24,7 @@ type ExitError interface {
 	Error() string
 	Exited() bool
 	ExitStatus() int
+	Unwrap() error
```

Comment:
this is breaking change to the interface, please 1/ check if this will break any existing code in the k/k repo 2/ send an email to kubernetes-sig-api-machinery@googlegroups.com to check with the community if this is is acceptable.

Reply:
@cfergeau did you already send this email that @yliaog mentioned?

Reply:
Nope, this totally slipped my mind, need to take another look at this when I have a bit of time.